docs/src/main/sphinx/admin/properties-optimizer.rst

```diff
@@ -146,3 +146,14 @@ Specifies minimal bucket to task ratio that has to be matched or exceeded in ord
 to use table scan node partitioning. When the table bucket count is small
 compared to the number of workers, then the table scan is distributed across
 all workers for improved parallelism.
+
+``colocated-joins-enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``boolean``
+* **Default value:** ``false``
+* **Session property:** ``colocated_join``
+
+*(Experimental)* Enables grouped join execution. If enabled, when both sides of
```

Comment:
It is flagged experimental in the code (see `show session`). This might be the reason it is not documented. If we remove experimental here we should also remove in the code. If we decide not to document due to it being experimental..

Reply:
Also ... there are comments in the code about enabling it by default. Where is that effort?

Reply:
I will defer to @sopel39 and @raunaqmorarka here.